backend/src/index.ts

```diff
@@ -50,12 +53,14 @@ app.get('/API/Elections', (req, res) => {
 
 // Create New Election
 app.post('/API/Elections', (req, res) => {
-    const Candidates = req.body.Election.Candidates.map((CandidateName:any,ind:any) => ({id: ind,CandidateName: CandidateName}))
-    const newElection = {
-        id: Elections.length,
-        ElectionName: req.body.Election.ElectionName,
-        Candidates: Candidates,
-    }
+    // const Candidates = req.body.Election.Candidates.map((CandidateName:any,ind:any) => ({id: ind,CandidateName: CandidateName}))
+    // const newElection = {
+    //     id: Elections.length,
+    //     ElectionName: req.body.Election.ElectionName,
+    //     Candidates: Candidates,
+    // }
+    const newElection = req.body.Election as Election;
+    newElection.electionId = String(Elections.length);
```

Comment:
Incremental ids is fine for now, but I wonder if we should consider random ids in the future?

Incremental shows people how many elections are on the site, and it makes it easy to iterate over all elections if they wanted to (which could be a privacy issue). I'm not sure if these issues matter to us (since we'll require auth to actually vote), but it could be worth thinking about more

Tom Scott goes into more details on the pros and cons at the end of this video: https://www.youtube.com/watch?v=gocwRvLhDf8

Reply:
Agreed, I think postgres would handle updating the ids? This is just a placeholder to store the data in memory until we get databases working.